src/JiraSecurityIssue.php

```diff
@@ -18,7 +19,8 @@ class JiraSecurityIssue
 {
     public const WATCHERS_TEXT = "This issue is being followed by %s";
     public const NO_WATCHERS_TEXT = "No watchers on this issue, remember to notify relevant people.";
-    public const NOT_FOUND_WATCHERS_TEXT = "Could not find user for %s, please check the users listed in JIRA_WATCHERS.";
+    public const NOT_FOUND_WATCHERS_TEXT =
+                                         "Could not find user for %s, please check the users listed in JIRA_WATCHERS.";
```

Comment:
I'd break the string into to and concat them in order to avoid this eyesore.

Reply:
Agree. I'll do that.